react-app/src/pages/Welcome.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component, Fragment } from "react";
+
+export class Welcome extends Component {
```

Comment:
remember, if you are not going to use local state or lifecycle methods => functional component